libs/common/src/helpers/date-validation.test.ts

```diff
@@ -0,0 +1,204 @@
+import { getHighestPriorityStandardDateErrorMessage } from './date-validation';
+
+describe('date-validation helpers', () => {
+  describe('getHighestPriorityStandardDateErrorMessage', () => {
+    const errorTestCases = [
+      {
+        description: 'Missing all values',
+        inputtedDay: null,
+        inputtedMonth: null,
+        inputtedYear: null,
+        expectedErrorMessage: 'Enter the Test name',
```

Comment:
Minor point, but I think if the name of the field is in the middle of the message it should not be capitalised

Reply:
(I avoided worrying about this before by starting error messages that start with the field name with 'The' and then passed in lower case only field name)